apis/management.cattle.io/v3/alerting_types.go

```diff
@@ -99,6 +99,155 @@ type AlertStatus struct {
 	AlertState string `json:"alertState,omitempty" norman:"options=active|inactive|alerting|muted,default=active"`
 }
 
+type ClusterAlertGroup struct {
```

Comment:
Where are all these types used? Are there rancher controllers that act on these?